app/scripts/controllers/meeting.status.ctrl.js

```diff
@@ -23,7 +23,7 @@ angular.module('dashboard')
         self.isEditing = false;
         self.unsavedConfig = { title: 'STR_CONFIRM', text: 'STR_WARNING_UNSAVED', yes: 'STR_CONTINUE' };
         var meetingItem = StorageSrv.getKey(CONST.KEY.MEETING_ITEM);
-        var isMobile = $rootScope.isMobile;
+        self.isMobile = $rootScope.isMobile;
```

Comment:
Why this is changed to public? Html uses directly root scope level isMobile function. Maybe html should be changed also or set this back to private...